commons/src/main/java/org/jetbrains/bsp/bazel/commons/Constants.java

```diff
@@ -64,5 +64,7 @@
 
   public static final String SCALA_TEST_MAIN_CLASSES_ATTRIBUTE_NAME = "main_class";
 
-  public static final String DEFAULT_PROJECT_VIEW_FILE = "projectview.bazelproject";
+  public static final String PROJECT_VIEW_FILE_PATH = "projectview.bazelproject";
+  public static final String DEFAULT_PROJECT_VIEW_FILE_PATH =
+      ".bazelbsp/default-projectview.bazelproject";
```

Comment:
Maybe define `default-projectview.bazelproject` in a central place once and reuse it in `BUILD`, `Install.java` and here.

Reply:
this file in general sucks and entire configuration should be done in a better way. In the following PR project view paths management will be changed anyway, because it will be configurable.